widget/check.go

```diff
@@ -337,9 +339,6 @@ func (c *checkRenderer) updateResource() {
 		bgRes.ColorName = theme.ColorNameBackground
 	}
 	if c.check.disabled.Load() {
-		if c.check.Checked {
-			res = theme.NewThemedResource(theme.CheckButtonCheckedIcon())
-		}
```

Comment:
For the reviewer: This was removed because it is already handled by the if-statement above.